### PR TITLE
Generate pdf

### PR DESCRIPTION
**Pull request by steven-jeanneret**
Work only in a synchronous way

### CODE REVIEW

File: app/File.php [hunk 1]
```diff
@@ -7,30 +7,64 @@
...
+subtitle: $this->subtitle
+date: $this->date
+school: $this->school
+papersize: a4
+geometry: margin=3cm
+titlepage: $this->is_title_page
+toc: $this->is_toc
+toc-own-page: $this->is_toc_own_page
+links-as-notes: $this->is_links_as_notes
+toc-depth: 5
```
Comment (greut) [inline app/File.php]: don't do that yourself... to risky.

http://php.net/manual/en/function.yaml-emit.php

Comment (steven-jeanneret) [reply app/File.php]: Done in last commit, with symfony/yaml (easier to install)

Comment (greut) [reply app/File.php]: `apt install php-yaml` ?

Comment (steven-jeanneret) [reply app/File.php]: This will work only on the server, symfony/yaml is installed by composer so it doesn't depend on the OS.

**Pull request merged**